sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/implementation/ManagementChannel.java

```diff
@@ -286,8 +287,8 @@
      * {@inheritDoc}
      */
     @Override
-    public Mono<Long> schedule(ServiceBusMessage message, Instant scheduledEnqueueTime, int maxLinkSize,
-        String associatedLinkName, ServiceBusTransactionContext transactionContext) {
+    public Mono<Long> schedule(ServiceBusMessage message, OffsetDateTime scheduledEnqueueTime, int maxLinkSize,
+                               String associatedLinkName, ServiceBusTransactionContext transactionContext) {
```

Comment:
nit: Do not need space This line would align with line below.

Reply:
Updated